postgresql/hooks/run

```diff
@@ -2,14 +2,23 @@
 
 set -e
 
+exec 2>&1
+
 echo "Starting PostgreSQL"
 
-export PGDATA={{pkg.svc_data_path}}
+export PGDATA=/hab/svc/postgresql/data
```

Comment:
Please don't hard code this, use the `{{}}` handlebars interpolation. The data dir in perticular may become configurable in a later hab version to support different mount points for persistent volumes.